camel-k/kafka-s3/kafka-not-secured-sink.kamelet.yaml

```diff
@@ -14,13 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ---------------------------------------------------------------------------
-apiVersion: camel.apache.org/v1alpha1
+apiVersion: camel.apache.org/v1
 kind: Kamelet
 metadata:
   name: kafka-not-secured-sink
   annotations:
     camel.apache.org/kamelet.support.level: "Stable"
-    camel.apache.org/catalog.version: "0.9.4"
+    camel.apache.org/catalog.version: "4.4.0"
```

Comment:
That's an example of camel-k, so the version of catalog should be compatible with the version of camel k and there is not 4.4.0 yet.

Reply:
I was thinking it would be the Kamelet catalog version which has the 4.4.0 https://github.com/apache/camel-kamelets/releases/tag/v4.4.0

So to which kind of version it corresponds?

there is no 0.9.4 tag for camel k https://github.com/apache/camel-k/tags?after=1.0.0-nightly.202003261646 , neither Camel K runtime https://github.com/apache/camel-k-runtime/tags?after=camel-k-runtime-parent-1.0.7

Reply:
The last release for 2.2.x is based on 4.0.1

https://camel.apache.org/camel-k/2.2.x/